test/UnitTests.proj

```diff
@@ -28,6 +28,7 @@
     <SDKCustomXUnitProject Condition="'$(RunAoTTests)' != 'true'" Include="..\src\Tasks\Microsoft.NET.Build.Tasks.UnitTests\Microsoft.NET.Build.Tasks.UnitTests.csproj" >
       <ExcludeAdditionalParameters>true</ExcludeAdditionalParameters>
     </SDKCustomXUnitProject>
+    <SDKCustomXUnitProject Condition="'$(RunAoTTests)' != 'true'" Include="dotnet-format.Tests\tests\dotnet-format.Unittests.csproj" />
```

Comment:
Add this to see if the tests of dotnet-fomat could be run on helix. I am not familar with this, so let me know if it needs more change.